source/_components/satel_integra.markdown

```diff
@@ -59,16 +61,20 @@ port:
   required: false
   default: 7094
   type: integer
-partition:
-  description: The partition to operate on. Integra can support multiple partitions, this platform only supports one.
-  required: false
-  default: 1
-  type: integer
-arm_home_mode:
-  description: The mode in which arm Satel Integra when 'arm home' is used. Possible options are `1`,`2` or `3`. For more information on what are the differences between them, please refer to Satel Integra manual.
-  required: false
-  default: 1
-  type: integer
+partitions:
+  description: List of the partitions to operate on.
+  required: true
+  type: [integer, list]
+  keys:
+    name:
+      description: Name of the partition.
+      required: true
+      type: string    
+    arm_home_mode:
+      description: The mode in which arm the partition when 'arm home' is used. Possible options are `1`,`2` or `3`. For more information on what are the differences between them, please refer to Satel Integra manual.
```

Comment:
The mode in which the partition is armed when...

Reply:
For more information on what the differences are between them...

Reply:
Fixed.